posts/2019/04/whats-new-in-sympy-14.md

```diff
@@ -0,0 +1,228 @@
+<!--
+.. title: What's New in SymPy 1.4
+.. slug: whats-new-in-sympy-14
+.. date: 2019-04-29
+.. author: Aaron Meurer
+.. tags: sympy, Labs, mathjax
+.. link:
+.. description:
+.. type: text
+.. has_math: yes
+-->
+
+As of November, 2018, I have been working at
+[Quansight](https://www.quansight.com/), under the heading of [Quansight
+Labs](https://www.quansight.com/labs). Quansight Labs is a public-benefit
+division of Quansight. It provides a home for a "PyData Core Team" which
+consists of developers, community managers, designers, and documentation
+writers who build open-source technology and grow open-source communities
+around all aspects of the AI and Data Science workflow. As a part of this, I
+am able to spend a fraction of my time working on SymPy.
+[SymPy](https://www.sympy.org/en/index.html), for those who do not know, is a
+symbolic mathematics library written in pure Python. I am the lead maintainer
+of SymPy.
+
+SymPy 1.4 was released on April 9, 2019. In this post, I'd like to go over
+some of the highlights for this release. The full release notes for the
+release can be found on the [SymPy
+wiki](https://github.com/sympy/sympy/wiki/Release-Notes-for-1.4).
+
+To update to SymPy 1.4, use
+
+```bash
+conda install sympy
+```
+
+or if you prefer to use pip
+
+```bash
+pip install -U sympy
+```
+
+The SymPy 1.4 release contains over [500 changes from 38 different
+submodules](https://github.com/sympy/sympy/wiki/Release-Notes-for-1.4#authors),
+so I will not be going over every change, but only a few of the main
+highlights. A [total of 104
+people](https://github.com/sympy/sympy/wiki/Release-Notes-for-1.4#authors)
+contributed to this release, of whom 66 contributed for the first time for
+this release.
+
+While I did not personally work on any of the changes listed below (my work
+for this release tended to be more invisible, behind the scenes fixes), I did
+do the release itself.
+
+# Automatic LaTeX rendering in the Jupyter notebook
+
+Prior to SymPy 1.4, SymPy expressions in the notebook rendered by default with their
+ string representation. To get `LaTeX` output, you had to call `init_printing()`:
+
+<img src="../../../../images/sympy-1.3-notebook.png" alt="SymPy 1.3 rendering in the Jupyter lab notebook">
```

Comment:
These relative links will be fragile I think. Better to use paths starting from the deployed root and use the markdown syntax:
```
![alt text](/images/filename.png)`

also, for my post I put images in `images/` not `files/images/`. I believe `files/` is for things that should be put in `/`, post images would be good to have in one place.